kernelci/runtime/__init__.py

```diff
@@ -125,13 +125,13 @@ def get_params(self, job, api_config=None):
         params = {
             'api_config': api_config or {},
             'storage_config': job.storage_config or {},
+            'platform_config': job.platform_config.to_dict() or {},
             'name': job.name,
             'node': job.node,
             'runtime': self.config.lab_type,
             'runtime_image': job.config.image,
         }
         params.update(job.config.params)
-        params.update(job.platform_config.params)
```

Comment:
I believe the idea was that this would enable platform configs to override any value already set.

Reply:
Same thing here.

Reply:
Tracked by #2170, fixed in #2171